src/Agent.Worker/ResourceMetricsManager.cs

```diff
@@ -97,15 +141,23 @@ public string GetCpuInfo()
             }
         }
 
-        public string GetMemoryInfo(ITerminal terminal)
+        public (int InstalledMemory, int UsedMemory, double MemoryUsageFree, double MemoryUsage) GetMemoryInfo()
```

Comment:
Isn't it better to define some class for it?

Reply:
Done. Thanks!